yarn-project/end-to-end/src/e2e_token_contract/transfer_in_private.test.ts

```diff
@@ -196,9 +196,7 @@ describe('e2e_token_contract transfer private', () => {
         .withWallet(wallets[1])
         .methods.transfer_in_private(badAccount.address, accounts[1].address, 0, nonce)
         .send();
-      await expect(txCancelledAuthwit.wait()).rejects.toThrow(
-        "Assertion failed: Message not authorized by account 'result == IS_VALID_SELECTOR'",
-      );
+      await expect(txCancelledAuthwit.wait()).rejects.toThrow('Assertion failed: Message not authorized by account');
```

Comment:
Not sure why I needed to do this change (was getting a failing test, see screenshot). But it seems inconsequential so would not worry about it. I think it might have something to do with us pruning the debug metadata from contracts artifacts or smt.

![image](https://github.com/user-attachments/assets/e7da4b5f-8907-4783-a644-afb0e0de4ab1)